docs/source/task_summary.rst

```diff
@@ -659,7 +661,7 @@ Here is an example of doing named entity recognition, using a model and a tokeni
 
 1. Instantiate a tokenizer and a model from the checkpoint name. The model is identified as a BERT model and loads it
    with the weights stored in the checkpoint.
-2. Define the label list with which the model was trained on.
+2. Define the label list with which the model was trained on, so that you can interpret the model's predictions.
```

Comment:
Adding more context here so that the purpose is clear. 

Question: Is this not available somehow in the model object itself?  Seems like it would be useful if it was!

Reply:
Good point, it's actually in the config: `model.config.id2label` gives the same thing as a dictionary.

Reply:
Awesome!  I went ahead and made this change.  Let me know what you think!